build/Targets/VSL.Imports.targets

```diff
@@ -468,4 +468,18 @@
                IntermediatesDirectory="$(IntermediateOutputPath)"
                Type="$(SignType)" />
   </Target>
+
+  <!-- ====================================================================================
+
+  Copying vsixmanifest to a separate folder
+
+  ==================================================================================== -->
+
+  <Target Name="AfterBuild" Condition="'$(CreateVsixContainer)' == 'true'">
+     <Copy SourceFiles="source.extension.vsixmanifest" DestinationFolder="$(OutDir)\Manifests\$(MSBuildProjectName)" ContinueOnError="true" />
```

Comment:
Why ContinueOnError? When would we never want to break the build?

This path is relative to what exactly? The project file or the targets?